### PR TITLE
Add rubygem-zest package

### PR DESCRIPTION
**Pull request by tbrisker**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.21
* [ ] 1.20

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: comps/comps-foreman-plugins-rhel7.xml [hunk 1]
```diff
@@ -158,7 +158,6 @@
       <packagereq type="default">rubygem-ruby-hmac-doc</packagereq>
       <packagereq type="default">rubygem-satyr-doc</packagereq>
       <packagereq type="default">rubygem-sequel-doc</packagereq>
-      <packagereq type="default">rubygem-smart-proxy-probing-doc</packagereq>
```
Comment (evgeni) [inline comps/comps-foreman-plugins-rhel7.xml]: The changes to this file seem unrelated?

File: comps/comps-foreman-rhel7.xml [hunk 2]
```diff
@@ -166,6 +166,7 @@
       <packagereq type="default">tfm-rubygem-statsd-instrument</packagereq>
       <packagereq type="default">tfm-rubygem-syntax</packagereq>
       <packagereq type="default">tfm-rubygem-text</packagereq>
+      <packagereq type="default">tfm-rubygem-typhoeus</packagereq>
```
Comment (evgeni) [inline comps/comps-foreman-rhel7.xml]: Should these be added to katello, not foreman comps?

Comment (tbrisker) [reply comps/comps-foreman-rhel7.xml]: fixed, appears like the add package script doesn't handle this correctly

**Comment by evgeni** [comment]
```
% koji add-pkg katello-nightly-rhel7 tfm-rubygem-typhoeus tfm-rubygem-zest --owner=kojiadm
Package tfm-rubygem-typhoeus already exists in tag katello-nightly-rhel7
Adding 1 packages to tag katello-nightly-rhel7
```

Interesting that typhoeus already existed?

**Comment by tbrisker** [comment]
:man_shrugging:
